src/test/scala/com/campudus/tableaux/TableauxTestBase.scala

```diff
@@ -20,19 +21,25 @@ case class TestCustomException(message: String, id: String, statusCode: Int) ext
 trait TableauxTestBase extends TestVerticle {
   val DEFAULT_PORT = 8181
 
+  lazy val config: JsonObject = jsonFromFile("../conf.json")
+  lazy val databaseAddress: String = config.getObject("database", Json.obj()).getString("address", "campudus.asyncdb")
```

Comment:
If we had an object with the default values, we could reuse it here:

```
object Starter {
  val DEFAULT_PORT = 8181
  val DEFAULT_DB_ADDRESS = "campudus.asyncdb"
}
```

and then have `Starter.DEFAULT_DB_ADDRESS`.